Quantum-Control-Applications-QuAM/Superconducting/quam_libs/components/flux_line.py

```diff
@@ -0,0 +1,39 @@
+from quam.components import SingleChannel
+from quam.components.ports import LFFEMAnalogOutputPort
```

Comment:
@nulinspiratie when toggling between OPX+ and LF fem, there is no need to change anything here right?
I can thus remove this line: `from quam.components.ports import LFFEMAnalogOutputPort`?

Reply:
It doesn't seem to be used in the file, so I would say yes it's fine to remove

Reply:
Yeah but what I mean is, if we now build quam with LF fem instead of OPX+, can we still use `SingleChannel` or do we need to change the component to be `FluxLine(LFFEMAnalogOutputPort`) instead?

Reply:
No, the channels are separated from the ports, so no changes needed

Reply:
It's a different matter for OPX + Octave versus MW FEM. Unfortunately there the changes are too significant between the two situations so it couldn't be combined into a single class